homeassistant/components/immich/coordinator.py

```diff
@@ -71,9 +73,12 @@ async def _async_update_data(self) -> ImmichData:
                 if self.is_admin
                 else None
             )
+            server_version_check = await self.api.server.async_get_version_check()
```

Comment:
Does this request work even if the version is too low?

Reply:
mhhh ... nice catch ... I don't think so